src/xma/include/lib/xmahw_lib.h

```diff
@@ -245,7 +234,7 @@ typedef struct XmaHwKernel
   XmaHwKernel() {
    std::memset(name, 0, sizeof(name));
     in_use = false;
-    context_opened = false;
+    context_opened = true;
```

Comment:
We need to make this context_opened variable to true when xrt::kernel object is created.

Reply:
context_opened is true by default and making it false when xrt::kernel creation failed, submitted these changes to the PR.